src/components/ApplicationDashboard.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react'
+import styled from 'styled-components'
+import { H1 } from './Typography'
+import { Button } from './Input/Button'
+import { SOCIAL_LINKS } from '../utility/Constants'
+import facebook from '../assets/icons/facebook.svg'
+import instagram from '../assets/icons/instagram.svg'
+import medium from '../assets/icons/medium.svg'
+import twitter from '../assets/icons/twitter.svg'
+import { ReactComponent as HandWave } from '../assets/hand-wave.svg'
+
+const Container = styled.div`
+  margin: 5em auto;
+  width: 70%;
```

Comment:
can we add a media query here and set width to 100% for mobile? the content looks too skinny and gets pushed down too much
<img width="312" alt="Screen Shot 2020-12-01 at 1 11 05 PM" src="https://user-images.githubusercontent.com/38872354/100797249-b05f8500-33d6-11eb-9607-741c71842961.png">
```suggestion
  width: 70%;
  ${maxWidthMediaQueries('mobile')} {
    margin: 0;
    width: 100%;
  }
```